cyberdrop_dl/crawlers/mydesi.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, ClassVar
+
+from cyberdrop_dl.crawlers.crawler import Crawler, SupportedPaths
+from cyberdrop_dl.data_structures import Resolution
+from cyberdrop_dl.data_structures.url_objects import AbsoluteHttpURL
+from cyberdrop_dl.utils import css
+from cyberdrop_dl.utils.utilities import error_handling_wrapper
+
+if TYPE_CHECKING:
+    from bs4 import BeautifulSoup
+
+    from cyberdrop_dl.data_structures.url_objects import ScrapeItem
+
+
+PRIMARY_URL = AbsoluteHttpURL("https://lolpol.com")
+
+
+class MyDesiCrawler(Crawler):
+    SUPPORTED_DOMAINS = "fry99.com", "lolpol.com", "mydesi.net"
+    SUPPORTED_PATHS: ClassVar[SupportedPaths] = {"Videos": "/title", "Search": "/search/"}
+    PRIMARY_URL: ClassVar[AbsoluteHttpURL] = PRIMARY_URL
+    DOMAIN: ClassVar[str] = "mydesi.net"
+    FOLDER_DOMAIN: ClassVar[str] = "MyDesi"
+
+    async def fetch(self, scrape_item: ScrapeItem) -> None:
+        if scrape_item.url.parts[1] == "search":
+            return await self.search(scrape_item)
+        if len(scrape_item.url.parts) == 2:
+            return await self.video(scrape_item)
+        raise ValueError
+
+    @error_handling_wrapper
+    async def video(self, scrape_item: ScrapeItem) -> None:
+        if await self.check_complete_from_referer(scrape_item):
+            return
+        soup = await self.request_soup(scrape_item.url)
+        resolution, download_url = self.select_highest_quality_video(soup)
+        title = soup.select_one("div.col-12.col-md-8.col-left > h1").text.strip()
+        filename, ext = self.get_filename_and_ext(download_url.name)
+
+        metadata_script = css.get_json_ld(soup)
+        upload_date = metadata_script.get("subjectOf", {}).get("uploadDate", "")
```

Comment:
The metadata has the clean title
```suggestion
        filename, ext = self.get_filename_and_ext(download_url.name)
        metadata = css.get_json_ld(soup)["subjectOf"]
        upload_date = metadata.get("uploadDate", "")
        title: str = metadata["name"]
```